modules/nf-core/svaba/meta.yml

```diff
@@ -0,0 +1,103 @@
+name: "svaba"
+description: SvABA is an efficient and accurate method for detecting SVs from short-read sequencing data using genome-wide local assembly with low memory and computing requirements
+keywords:
+  - sv
+  - structural variants
+  - detecting svs
+  - short-read sequencing
+tools:
+  - "svaba":
+      description: "Structural variation and indel detection by local assembly"
+      homepage: "https://github.com/walaj/svaba"
+      documentation: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880247/"
+      tool_dev_url: "https://github.com/walaj/svaba"
+      doi: "10.1101/gr.221028.117"
+      licence: "['GPL v3']"
+
+input:
+  - meta:
```

Comment:
Can you also add the other meta fields here?

Reply:
done